crates/apollo-compiler/src/lib.rs

```diff
@@ -103,6 +103,20 @@ impl ApolloCompiler {
         file_id
     }
 
+    // This adds the introspection type system and any built-in graphql types.
+    fn add_implicit_types(&mut self) {
+        let path = Path::new("src/built_in_types.graphql");
+        if self.db.source_file(path.to_path_buf()).is_none() {
+            let file_id = FileId::new();
+            let mut sources = self.db.source_files();
+            sources.push(file_id);
+            let implicit_tys = fs::read_to_string(path).expect("built in types could not be added");
```

Comment:
I think this expect the process current directory to be the apollo-compiler source tree, which typically won’t be the case when compiler is used as library dependency. Consider using `include_str!` instead